examples/proto/examplepb/a_bit_of_everything.proto

```diff
@@ -94,13 +94,45 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
 			value: {};
 		}
 	}
+	responses: {
+		key: "403";
+		value: {
+			description: "Returned when the user does not have permission to access the resource.";
+		}
+	}
+	responses: {
+		key: "404";
+		value: {
+			description: "Returned when the resource does not exist.";
+			schema: {
+				json_schema: {
+					type: STRING;
+				}
+			}
+		}
+	}
+	responses: {
+		key: "418";
+		value: {
+			description: "I'm a teapot.";
+			schema: {
+				json_schema: {
+					ref: ".grpc.gateway.examples.examplepb.NumericEnum";
+				}
+			}
+		}
+	}
 };
 
 
 // Intentionaly complicated message type to cover much features of Protobuf.
 // NEXT ID: 30
 message ABitOfEverything {
 	option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+		json_schema: {
+			title: "A bit of everything"
+			description: "Intentionaly complicated message type to cover much features of Protobuf."
```

Comment:
Nit: s/much/many/

We can skip this if it'd take you a lot of time to regenerate examples.